aospy/calc.py

```diff
@@ -767,3 +768,23 @@ def load(self, dtype_out_time, dtype_out_vert=False, region=False,
         if plot_units:
             data = self.var.to_plot_units(data, dtype_vert=dtype_out_vert)
         return data
+
+def _add_units_and_description_attrs(data, units, description, dtype_out_vert):
+    if isinstance(data, xr.DataArray):
+        _add_units_and_description_attrs_da(data, units, description,
+                                            dtype_out_vert)
+    else:
+        for name, da in data.data_vars.items():
+            _add_units_and_description_attrs_da(da, units, description,
+                                                dtype_out_vert)
+
+def _add_units_and_description_attrs_da(data, units, description,
+                                        dtype_out_vert):
```

Comment:
Both of these helper functions should have a one-line docstring.  (Since they're private API and pretty self explanatory, a full one listing parameters, return value, etc., isn't necessary.)